cobbler.spec

```diff
@@ -264,16 +266,23 @@ mkdir -p %{buildroot}%{tftpboot_dir}/{boot,etc,grub,images{,2},ppc,pxelinux.cfg,
 # systemd
 mkdir -p %{buildroot}%{_unitdir}
 mv %{buildroot}%{_sysconfdir}/cobbler/cobblerd.service %{buildroot}%{_unitdir}
+ln -sf service %{buildroot}%{_sbindir}/rccobblerd
```

Comment:
This will _only_ work on SUSE distributions, so don't create the symlink for anything else. You'll just create a broken program invocation.

Reply:
Done